ci-operator/step-registry/cucushift/upgrade/setchannel/cucushift-upgrade-setchannel-commands.sh

```diff
@@ -60,17 +43,16 @@ if [[ -f "${SHARED_DIR}/proxy-conf.sh" ]]; then
     source "${SHARED_DIR}/proxy-conf.sh"
 fi
 
-mkdir -p /tmp/client
-export OC_DIR="/tmp/client"
-export PATH=${OC_DIR}:$PATH
+# oc cli is injected from release:target
+run_command "which oc"
+run_command "oc version --client"
+
 export DUMMY_TARGET="${OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE}"
-# we need this DUMMY_TARGET_VERSION from ci config to download oc
 DUMMY_TARGET_VERSION="$(env "NO_PROXY=*" "no_proxy=*" oc adm release info "${DUMMY_TARGET}" --output=json | jq -r '.metadata.version')"
 echo "Target version of ci config is: ${DUMMY_TARGET_VERSION}"
 x_ver=$( echo "${DUMMY_TARGET_VERSION}" | cut -f1 -d. )
 y_ver=$( echo "${DUMMY_TARGET_VERSION}" | cut -f2 -d. )
 export y_ver
```

Comment:
nit: this is added for extract_oc, it could be removed together.

Reply:
Done